### DISCUSSION TITLE
GHSA-f96h-pmfr-66vw affecting starlette <0.39.2

### DISCUSSION THREAD

**Discussion by jkmcclinton** [fastapi/fastapi#12480] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
n/a
```

### Description

This vulnerability affects starlette < 0.39.2, and fastapi 0.115.2 requires starlette < 0.41.0 and >= 0.37.2. Is there an ETA on when this will get resolved? Without pinning the version, pip installs the vulnerable 0.37.2.

Thanks

### Operating System

Linux, Windows

### FastAPI Version

0.115.2

### Pydantic Version

2.8.2

### Python Version

3.11

**Comment by YuriiMotov** [comment]
Resolved in v 0.115.3: https://fastapi.tiangolo.com/release-notes/#01153
